src/main/java/Duke.java

```diff
@@ -1,10 +1,111 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
+	public static void main (String[] args) {
+		printStart ();
```

Comment:
I think this is a very clean way to print the start messages, and keeps the main method clean